src/main/java/duke/main/Ui.java

```diff
@@ -0,0 +1,92 @@
+package duke.main;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    private Scanner sc;
+
+    Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Prints a greeting message when the bot is opened.
+     */
+    public void greetingMessage() {
+        String greeting = "Hello! I'm Duke\n\t" +
+                "What can I do for you?";
+        printMessage(greeting);
+    }
+
+    /**
+     * Prints an exit message.
+     */
+    public void exitMessage() {
```

Comment:
Maybe this method name could also be renamed as showExitMessage() to sound more like a verb?